### PR TITLE
Lf2268/add more items button

### PR DESCRIPTION
**Pull request by MwayaB**

### CODE REVIEW

File: packages/webapp/src/containers/Finances/NewExpense/AddExpense/index.jsx [hunk 1]
```diff
@@ -200,10 +205,11 @@ class AddExpense extends Component {
                     ))}
                   </div>
                   <div className={styles.addContainer}>
-                    <div className={styles.greenPlus}>+</div>
-                    <button onClick={() => this.addSubExpense(k)}>
-                      {this.props.t('EXPENSE.ADD_MORE_ITEMS')}
-                    </button>
+                    <Tooltip title={this.props.t('EXPENSE.ADD_MORE_ITEMS')}>
+                      <div className={styles.greenPlus} onClick={() => this.addSubExpense(k)}>
+                        +
```
Comment (smattingly) [inline packages/webapp/src/containers/Finances/NewExpense/AddExpense/index.jsx:210]: The text no longer appears on the page-- just as the tool tip. I don't think the ticket said anything about it. Was it intentional?

File: packages/webapp/src/containers/Finances/NewExpense/AddExpense/index.jsx [hunk 2]
```diff
@@ -200,10 +205,11 @@ class AddExpense extends Component {
                     ))}
                   </div>
                   <div className={styles.addContainer}>
-                    <div className={styles.greenPlus}>+</div>
-                    <button onClick={() => this.addSubExpense(k)}>
-                      {this.props.t('EXPENSE.ADD_MORE_ITEMS')}
-                    </button>
+                    <Tooltip title={this.props.t('EXPENSE.ADD_MORE_ITEMS')}>
+                      <div className={styles.greenPlus} onClick={() => this.addSubExpense(k)}>
```
Comment (smattingly) [inline packages/webapp/src/containers/Finances/NewExpense/AddExpense/index.jsx:209]: The ticket says to add new button style. I don't think that has happened?

**Review by smattingly — CHANGES_REQUESTED** [review]
Had a couple of questions about what the intended results are.

File: packages/webapp/src/containers/Finances/NewExpense/AddExpense/index.jsx (hunk 1 above)
Comment (MwayaB) [reply packages/webapp/src/containers/Finances/NewExpense/AddExpense/index.jsx:210]: Yes it was intentional, no the ticket doesn't say that, I just thought the text looked odd and a tooltip would work better. I should have consulted with Sasha first.

File: packages/webapp/src/containers/Finances/NewExpense/AddExpense/index.jsx (hunk 2 above)
Comment (MwayaB) [reply packages/webapp/src/containers/Finances/NewExpense/AddExpense/index.jsx:209]: I need to consult with Sasha.

**Review by smattingly — APPROVED** [review]
:+1:

**Pull request merged**